packages/react-composites/src/composites/CallComposite/components/Prompt.tsx

```diff
@@ -27,16 +28,18 @@ export interface PromptProps {
   cancelButtonLabel?: string;
   onConfirm?: () => void;
   onCancel?: () => void;
+  styles?: Partial<IModalStyles>;
 }
 
 /**
  * @private
  */
 export const Prompt = (props: PromptProps): JSX.Element => {
   const theme = useTheme();
+  const styles = props.styles ?? modalStyles;
```

Comment:
nit: I think we should change this to `const styles = concatStyles(modalStyles, props.styles)` just so not all default css properties are overridden when you assign styles. But you can still override all default styles.